docs/sources/enterprise/access-control/permissions.md

```diff
@@ -21,55 +21,57 @@ scope
-roles:read | roles:* | Allows to read a specific role with it's permissions.
-roles:write | permissions:delegate | Allows to create or update a custom role.
-roles:delete | permissions:delegate | Allows to delete a custom role.
-roles.builtin:list | roles:* | Allows to list built-in role assignments.
-roles.builtin:add | permissions:delegate | Allows to create a built-in role assignment.
-roles.builtin:remove | permissions:delegate | Allows to delete a built-in role assignment.
-reports.admin:write | reports:* | Allows to create or update reports.
-reports:delete | reports:* | Allows to delete reports.
-reports:read | reports:* | Allows to list all available reports and to get a specific report. 
-reports:send | reports:* | Allows to send report email.
-reports.settings:write | n/a | Allows to update report settings.
-reports.settings:read | n/a | Allows to read report settings.
-provisioning:reload | service:access-control | Allows to reload provisioning files after an update.
-users:read | global:users:* | Allows to read, search user profiles. 
-users:write | global:users:* | Allows to update user profiles.
-users.teams:read | global:users:* | Allows to read user teams.
-users.authtoken:list | global:users:* | Allows to list auth tokens assigned to users.
-users.authtoken:update | global:users:* | Allows to update auth tokens assigned to users.
-users.password:update | global:users:* | Allows to update users password.
-users:delete | global:users:* | Allows to delete users.
-users:create | n/a | Allows to create users.
-users:enable | global:users:* | Allows to enable users.
-users:disable | global:users:* | Allows to disable users.
-users.permissions:update | global:users:* | Allows to update users org level permissions.
-users:logout | global:users:* | Allows to enforce logout for users.
-users.quotas:list | global:users:* | Allows to list user quotas.
-users.quotas:update | global:users:* | Allows to update user quotas.
-org.users.read | users:* | Allows to get user profiles within the organization.
-org.users.add | users:* | Allows to add users to the organization.
-org.users.remove | users:* | Allows to remove users from the organization.
-org.users.role:update | users:* | Allows to update users organization role for the assigned organization.
-ldap.user:read | n/a | Allows to read LDAP users.
-ldap.user:sync | n/a | Allows to sync LDAP users.
-ldap.status:read | n/a | Allows to check LDAP status.
+roles:list | roles:* | List available roles without permissions.
+roles:read | roles:* | Read a specific role with it's permissions.
+roles:write | permissions:delegate | Create or update a custom role.
+roles:delete | permissions:delegate | Delete a custom role.
+roles.builtin:list | roles:* | List built-in role assignments.
+roles.builtin:add | permissions:delegate | Create a built-in role assignment.
+roles.builtin:remove | permissions:delegate | Delete a built-in role assignment.
+reports.admin:create | reports:* | Create reports.
```

Comment:
Need to add this the concerned HTTP endpoint

Reply:
I cannot find the `POST /api/reports/new` So I assume we can skip that

Reply:
This endpoint is not documented in the HTTP API in the reporting page it just mentioned you need to have the right permission. For now having this mentioned in list of actions is sufficient